src/app/canvas/Canvas.tsx

```diff
@@ -0,0 +1,128 @@
+'use client';
+
+import { useState, useCallback } from 'react';
+import update from 'immutability-helper';
+import { Stack } from '@mui/material';
+import { TaskCard } from '@/components/cards/TaskCard';
+import { TaskColumn } from '@/components/cards/TaskColumn';
+import { Droppable } from '@/hoc/Droppable';
+
+interface Task {
```

Comment:
I recommend moving the type and mock data to a separate file. That file will be replaced by the logic that loads this data and that is also where the type would be defined

Reply:
Sure. It is only for now I have kept all the mock data and type defs on the same page. The type defs will depend on the API response and while working on the API integration I should remove the mock data and add new type defs that should be compatible with the response from the API.